code/game/machinery/computer/launchpad_control.dm

```diff
@@ -38,14 +139,17 @@
 		return FALSE
 	return TRUE
 
-/obj/machinery/computer/launchpad/proc/teleport(mob/user, obj/machinery/launchpad/pad, sending)
+/obj/machinery/computer/launchpad/proc/teleport_checks(obj/machinery/launchpad/pad)
```

Comment:
```suggestion
/// Performs checks on whether or not the launch pad can be used.
/// Returns `null` if there are no errors, otherwise will return the error string.
/obj/machinery/computer/launchpad/proc/teleport_checks(obj/machinery/launchpad/pad)
```